functions.php

```diff
@@ -31,7 +31,10 @@ function wp_auth0_get_option( $key, $default = null ) {
 function wp_auth0_is_current_login_action( array $actions ) {
 
 	// Not on wp-login.php.
-	if ( ! isset( $GLOBALS['pagenow'] ) || 'wp-login.php' !== $GLOBALS['pagenow'] ) {
+	if (
+		( isset( $GLOBALS['pagenow'] ) && 'wp-login.php' !== $GLOBALS['pagenow'] ) &&
```

Comment:
no longer an OR, is this desired?

Reply:
This is desired. Need to either have that page set to `wp-login.php` or that function. If the page has the function, it will not be the wp-login.php `pagenow`